### ISSUE TITLE
Native video download feature

### ISSUE THREAD

**Issue by Danoloan10** [zedeus/nitter#263]
I mostly use nitter on my phone, and it would be awesome if I could download videos directly from the page, as I would do with an image. I've seen #153 , but it's not feasible in a phone.

**Comment by aviegdown** [comment]
![Screenshot_2020-12-18-08-11-53-121_org mozilla fenix](https://user-images.githubusercontent.com/72583750/102562009-b9579280-4108-11eb-9096-429a497179ee.jpg)
i do able to download directly using firefox browser

**Comment by Danoloan10** [comment]
i'm using bromite browser. i think the problem is that bromite doesn't support download of hls streams.

![Screenshot_20201218-115014_Bromite](https://user-images.githubusercontent.com/8666137/102606588-3c87e100-4127-11eb-9b4b-5c98e767a09f.png)

there is no option for download, on a normal mp4 stream there would be three dots down to the right of the video with that option. 

I wonder if this is the case of other browsers too.

**Comment by Mennaruuk** [comment]
@Danoloan10 Correct. In fact, Bromite can't even play .m3u8 files. Someone suggested this be integrated into Bromite, and it was [rejected](https://github.com/bromite/bromite/issues/888).

You can try playing it on another browser such as Brave or Firefox.

**Comment by Thespikedballofdoom** [comment]
I also want to be able to DL stuff, right from the website in firefox...

**Comment by garret** [comment]
I have both Bromite and Fennec installed on my Pixel 6 (Android 12). However, with both browsers, I cannot find an option to download videos. It has been previously written that it should be possible at least with Firefox, but don't understand how to get a "save video" option. Have been trying to tap on the video and even long tapping. Using nitter as main instance but also with others have the same issue. 

![Screenshot_20220705-125009](https://user-images.githubusercontent.com/27555/177312457-78510767-46e8-4554-857e-4d83390b238c.png)

**Comment by zedeus** [comment]
It's not supposed to work right now @garret, so that's expected.
